pkgs/top-level/all-packages.nix

```diff
@@ -25931,4 +25931,5 @@ in
 
   quartus-prime-lite = callPackage ../applications/editors/quartus-prime {};
 
+  drawing = callPackage ../applications/graphics/drawing { };
```

Comment:
Please move it to the appropriate section and in alphabetical order

Reply:
moved